about/release_policy.rst

```diff
@@ -158,3 +195,55 @@ The 3.6 release is still planned and should be the last stable branch of Godot
 as long as users still need it (due to missing features in Godot 4.x, or
 having published games which they need to keep updating for platform
 requirements).
+
+What are the criteria for compatibility across engine versions?
+---------------------------------------------------------------
+
+.. note::
+
+    This section is intended to be used by contributors to determine which
+    changes are safe for a given release. The list is not exhaustive; it only
+    outlines the most common situations encountered during Godot's development.
+
+The following changes are acceptable in patch releases:
+
+- Fixing a bug that has no major negative impact on most projects, such as a
+  visual or physics bug. Godot's physics engine is not deterministic, so physics
+  bug fixes are not considered to break compatibility. If fixing a bug has a
+  negative impact that could impact a lot of projects, it should be made
+  optional (e.g. using a project setting or separate method).
+- Adding a new optional parameter to a method.
+- Small-scale editor usability tweaks.
+
+The following changes are acceptable in minor releases, but not patch releases:
+
+- Significant new features.
+- Renaming a method parameter. In C#, method parameters can be passed by name
+  (but not in GDScript). As a result, this can break some projects that use C#.
+- Deprecating a method, member variable or class. This is done by adding a
+  deprecated flag to its class reference, which will show up in the editor. When
+  a method is marked as deprecated, it's slated to be removed in the next
+  *major* release.
+- Changes that affect the default project theme's visuals.
+- Performance optimizations that result in visual changes.
+
+The following changes are considered **compatibility-breaking** and can only be
+performed in a new major release:
+
+- Renaming or removing a method, member variable or a class.
+- Modifying a node's inheritance tree by making it inherit from a different class.
```

Comment:
```suggestion
- Modifying node's inheritance tree by making it inherit from a different class.
```

Reply:
Isn't "node" a countable noun?

Reply:
It is. I'm uncertain about the rules of using articles around possessive nouns, but after a bit of searching I think you've got it correct originally.